### PR TITLE
fix: make contract more secure. Update deps and fix ci test

### PR DESCRIPTION
**Pull request by freesig**

### CODE REVIEW

File: apps/utils/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "essential-app-utils"
 description = "Utilities for building Essential applications"
-version = "0.4.0"
+version = "0.5.0"
```
Comment (supiket) [inline apps/utils/Cargo.toml:4]: Looks like the version on crates.io is already 0.5.0 🤔

File: apps/utils/src/builder.rs [hunk 2]
```diff
@@ -4,7 +4,9 @@ use essential_types::{solution::Solution, ContentAddress};
 use crate::db::Dbs;
 
 pub async fn build_default(dbs: &Dbs) -> Result<SolutionsSummary, BuildBlockError> {
-    essential_builder::build_block_fifo(&dbs.builder, &dbs.node, &Default::default()).await
+    essential_builder::build_block_fifo(&dbs.builder, &dbs.node, &Default::default())
+        .await
+        .map(|(_, o)| o)
```
Comment (supiket) [inline apps/utils/src/builder.rs:9]: Accidental emoticon

**Review by supiket — APPROVED** [review]
Looks great, closes #100

File: apps/utils/Cargo.toml (hunk 1 above)
Comment (freesig) [reply apps/utils/Cargo.toml:4]: Yeh I had to manually publish to get this PR to pass

**Pull request merged**